src/util.ts

```diff
@@ -101,13 +103,23 @@ export function isCacheFeatureAvailable(): boolean {
 }
 
 export function getVersionFromFileContent(
+  fileName: string,
```

Comment:
Since the behavior of this function depends on the filename I suggest changing the name to `getVersionFromFile`.